app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.jsx

```diff
@@ -244,69 +218,95 @@ export const Notifications = ({ setHeaderTitleNode }) => {
     }
   };
 
-  const handleDocumentationClick = () => {
-    trackEvent(PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.clickDocumentationLink('no_notifications'));
-  };
-
   if (loading) {
     return <SpinningPreloader />;
   }
+  // separate notifications by types
+  const notificationRulesByTypes = Object.groupBy(notifications, (rule) => rule.type);
 
   return (
-    <>
-      {notifications.length ? (
-        <SettingsPageContent>
-          <Layout
-            description={
-              <FormattedDescription
-                content={formatMessage(messages.tabDescription, {
-                  a: (data) => createExternalLink(data, docsReferences.notificationsDocs),
-                })}
-                event={PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.clickDocumentationLink()}
-              />
-            }
-          >
+    <SettingsPageContent>
+      <Layout
+        description={<FormattedDescription content={formatMessage(messages.tabDescription)} />}
+      >
+        <FieldElement
+          className={cx('fieldElement')}
+          withoutProvider
+          description={formatMessage(messages.toggleNote)}
+          dataAutomationId="notificationsEnabledCheckbox"
+        >
+          <div className={cx('toggle')}>
+            <Toggle
+              disabled={isReadOnly}
+              value={enabled}
+              onChange={(e) => toggleNotificationsEnabled(e.target.checked)}
+              dataAutomationId="enabledToggle"
+            >
+              <span className={cx('name-wrapper')}>
+                <i className={cx('name')}> {formatMessage(messages.allNotifications)}</i>
+              </span>
+            </Toggle>
+          </div>
+        </FieldElement>
+      </Layout>
+      {/* render extension: including name, description and subRules */}
+      {allNotificationPlugins.map((item, id) => (
+        // eslint-disable-next-line react/no-array-index-key
+        <div className={cx('rule-section')} key={`rule-section-${id}`}>
```

Comment:
If the names in the plugins are all different, then you can make a unique value and remove the comment:
`rule-section-${item.name}` 
or `componentName-elementName-${uniqueValue}` => `notifications-rule-section-${item.name}`